test/Data.UnitTests/Detection/DetectionFactoryTests.cs

```diff
@@ -26,17 +29,32 @@ public void Factory_ShouldConvert_Entity()
             _model.ShouldBeEquivalentTo(outputDetection);
         }
 
+        [Test]
+        public void Factory_shouldConvert_MCreateDetection()
+        {
+            var outputDetection = DetectionFactory.MakeEntity(_modelCreate);
+
+            _entity.ShouldBeEquivalentTo(outputDetection);
+        }
+
         private static EDetection CreateEntity() => new ()
         {
-            Id = 1,
+            Id = 0,
+            Class = "Class",
+            Score = 100,
+            Timestamp = new DateTime(2021, 07, 20),
+        };
+
+        private static MCreateDetection CreateDetection() => new ()
```

Comment:
Would the name `CreateInput` be more relevant here?

Reply:
Done